TUTORIAL.md

```diff
@@ -360,26 +360,22 @@ First, define where CesiumJS is. This tutorial uses the source code, so webpack
    import {
      Ion,
      Viewer,
-     createWorldTerrain,
-     createOsmBuildings,
+     Terrain,
+     createOsmBuildingsAsync,
      Cartesian3,
      Math,
    } from "cesium";
    import "cesium/Widgets/widgets.css";
    import "../src/css/main.css";
 
-   // Your access token can be found at: https://cesium.com/ion/tokens.
-   // This is the default access token
-   Ion.defaultAccessToken =
-     "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWE1OWUxNy1mMWZiLTQzYjYtYTQ0OS1kMWFjYmFkNjc5YzciLCJpZCI6NTc3MzMsImlhdCI6MTYyNzg0NTE4Mn0.XcKpgANiY19MC4bdFUXMVEBToBmqS8kuYpUlxJHYZxk";
```

Comment:
Both of the examples actual code includes this line commented out to make it clear the user should replace it with their own. Maybe we continue that here?

```js
// CesiumJS has a default access token built in but it's not meant for active use.
// please set your own access token can be found at: https://cesium.com/ion/tokens.
// Ion.defaultAccessToken = "YOUR TOKEN HERE";
```